# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tse3/tse3_type.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- tse3-0.2.7/src/tse3/file/Track.cpp.orig	2005-08-23 15:34:25.000000000 -0700
+++ tse3-0.2.7/src/tse3/file/Track.cpp	2005-08-23 15:13:04.000000000 -0700
@@ -13,7 +13,7 @@
     write(writer, *t.filter());
     write(writer, *t.params());
     write(writer, *t.displayParams());
-    writer.element("NoParts", t.size());
+    writer.element("NoParts", (unsigned int)t.size());
 
     for (size_t n = 0; n < t.size(); ++n)
     {
--- tse3-0.2.7/src/tse3/file/Song.cpp.orig	2005-08-23 15:35:22.000000000 -0700
+++ tse3-0.2.7/src/tse3/file/Song.cpp	2005-08-23 15:13:03.000000000 -0700
@@ -15,7 +15,7 @@
     writer.element("Author",    song.author());
     writer.element("Copyright", song.copyright());
     writer.element("Date",      song.date());
-    writer.element("NoTracks",  song.size());
+    writer.element("NoTracks",  (unsigned int)song.size());
 
     writer.comment("Master tracks");
     write(writer, *song.tempoTrack());
